### PR TITLE
chore: update copyrights years

### PR DESCRIPTION
**Pull request by agnes512**
#### Related issues

- connect to https://github.com/strongloop-internal/scrum-apex/issues/406

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

File: example/example.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
 // Copyright IBM Corp. 2016. All Rights Reserved.
 // Node module: loopback-connector-informix
-// This file is licensed under the Artistic License 2.0.
```
Comment (dhmlau) [inline example/example.js]: I wonder why these 2 lines are removed.  
For example https://github.com/strongloop/loopback-connector-mysql/blob/master/lib/discovery.js, it has the snippet about "this file is licensed under....".

Comment (nabdelgadir) [reply example/example.js]: They're removed because in package.json, the `license` is `"SEE LICENSE IN LICENSE"` instead of `Artistic License 2.0`.

Comment (agnes512) [reply example/example.js]: added them back.

File: test/describe.js [hunk 2]
```diff
@@ -1,3 +1,6 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: loopback-connector-informix
+
```
Comment (nabdelgadir) [inline test/describe.js]: Can you add the two lines here too?

**Pull request merged**